src/widgets.jl

```diff
@@ -445,8 +445,9 @@ function textbox(::Type{T};
     end
     set_gtk_property!(widget, "text", value)
 
-    id = signal_connect(widget, gtksignal) do w
+    id = signal_connect(widget, gtksignal) do w, _...
         setindex!(observable, entrygetter(w, observable, range))
+        return false
```

Comment:
Do we need to make this change elsewhere in the package or is this the only place?

Reply:
Textarea perhaps?

Reply:
Oh I guess textarea uses the `changed` event by default, so no.